CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased
+
+* Adds `BugReporting.setVideoRecordingFloatingButtonPosition` API
```

Comment:
I think we just need to remove the backticks for now if that'd cause any issues publishing the changelog on the dashboard. Good job with the PR :)

Reply:
Thanks, David 🙏🏼. I resolved that.